src/components/Image/types.ts

```diff
@@ -0,0 +1,50 @@
+import {ImageRequireSource, ImageResizeMode, ImageStyle, ImageURISource, StyleProp} from 'react-native';
+import {ImageStyle as FastImageStyle, OnLoadEvent, ResizeMode, Source} from 'react-native-fast-image';
+import {OnyxEntry} from 'react-native-onyx';
+import {Session} from '@src/types/onyx';
+
+type ImageOnyxProps = {
+    /* Onyx Props */
+    /** Session info for the currently logged in user. */
+    session: OnyxEntry<Session>;
+};
+
+type ImageProps = {
```

Comment:
```suggestion
type ImageProps = ImageOnyxProps & {
```

I think `ImageProps` should already be this type definition with `ImageOnyxProps`, and then we remove `ImagePropsWithOnyx` and replace its usage with `ImageProps`.

Reply:
Updated!